src/Core/Silk.NET.SilkTouch/INativeContextOverride.cs

```diff
@@ -3,12 +3,13 @@
 
 using System;
 using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 
 namespace Silk.NET.SilkTouch
 {
     public interface INativeContextOverride
     {
-        TypeDeclarationSyntax Type(string name, string lib, EntryPoint[] entrypoints);
+        TypeDeclarationSyntax Type(string name, string lib, EntryPoint[] entrypoints, Compilation comp);
```

Comment:
That doesn't seem right

Reply:
Changed to take a new context structure.